src/object_detection.cpp

```diff
@@ -1,35 +1,39 @@
 #include <ros/ros.h>
-#include <sensor_msgs/Image.h>
-#include <object_detection_pkg/ObjDetected.h>
 #include <ros/console.h>
+#include <ros/package.h>
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/image_encodings.h>
 #include <geometry_msgs/Vector3.h>
```

Comment:
I don't think this header is being used, can remove if so!

Reply:
Sounds good, I removed them and made sure it builds.